words/predicates.txt

```diff
@@ -112,20 +128,26 @@ bristle
 broad
 broken
 bronze
+bronzed
 brook
 bubble
+bubbly
 bumpy
 burly
 bush
+bushy
 bustling
 busy
 butter
 buttercup
+buttered
```

Comment:
"buttered muffin" was supposed to be a dirty joke on Mean Girls but I'm not sure if anyone actually uses that term?

Reply:
yeah, they just made up the phrase for mean girls in order to sound dirty but not actually mean anything :D

Reply:
Today I Learned!